openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -265,6 +265,11 @@ jQuery(function () {
             .then((module) => module.initAdmin());
     }
 
+    if (window.matchMedia('(display-mode: standalone)').matches) {
+        import(/* webpackChunkName: "offline-banner" */ './offline-banner')
+            .then((module) => module.initOfflineBanner());
```

Comment:
Let's rename this file to pwa-extras; it will likely have more small helpers in the future.

```suggestion
        import(/* webpackChunkName: "pwa-extras" */ './pwa-extras')
            .then((module) => module.init());
```